### PR TITLE
[TS-SDK] Add enums in `getNormalizedMoveModule`

### PR DESCRIPTION
**Pull request by vladilen11**
## Description 

Adds enums to the NormalizedMoveModule for TS-SDK.

## Test plan 

How did you test the new or updated feature?

---

## Release notes

Check each box that your changes affect. If none of the boxes relate to your changes, release notes aren't required.

For each box you select, include information after the relevant heading that describes the impact of your changes that a user might notice and any actions they must take to implement updates. 

- [ ] Protocol: 
- [ ] Nodes (Validators and Full nodes): 
- [ ] Indexer: 
- [x] JSON-RPC: https://github.com/MystenLabs/sui/pull/20231
- [ ] GraphQL: 
- [ ] CLI: 
- [ ] Rust SDK:
- [ ] REST API:
- [x] TS SDK: Fixes an issue where enum types were not included in the output of getNormalizedMoveModule and getNormalizedMoveModuleForPackage

related issue: 
- https://github.com/MystenLabs/sui/issues/20205

### CODE REVIEW

**Review by hayes-mysten — APPROVED** [review]

**Comment by hayes-mysten** [comment]
It looks like this creates an error in the graphql-transport pacakge

**Review by hayes-mysten — APPROVED** [review]
These types are generated based on the openrpc spec, so I fixed a small issue in the generator, and re-ran it rather than updating the types manually

File: sdk/graphql-transport/src/mappers/move.ts [hunk 1]
```diff
@@ -156,6 +157,7 @@ export function mapNormalizedMoveModule(
 ): SuiMoveNormalizedModule {
 	const exposedFunctions: Record<string, SuiMoveNormalizedFunction> = {};
 	const structs: Record<string, SuiMoveNormalizedStruct> = {};
+	const enums: Record<string, SuiMoveNormalizedEnum> = {}; // TODO: Implement
```
Comment (hayes-mysten) [inline sdk/graphql-transport/src/mappers/move.ts:160]: It looks like enums are actually available in the GraphQL api now, but this requires the graphql client to be updated.  The GraphQL release process has changed, and updating the client will take a bit of additional work.  So this should be fine for now, until we get the client updated

**Comment by vladilen11** [comment]
Hi @hayes-mysten , just want ask when will client be updated? We're waiting for the enum support update. 😄

**Comment by hayes-mysten** [comment]
Ran into some issues getting this updated on this branch, I've opened up a new PR for this here: https://github.com/MystenLabs/sui/pull/20684

**Pull request closed without merging**